### PR TITLE
Add Helix Dockerfile for Fedora 29

### PR DESCRIPTION
**Pull request by MattGal**

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -178,6 +178,17 @@
             }
           ]
         },
+        {
+          "platforms": [
+            {
+              "dockerfile": "src/fedora/29/helix/amd64",
+              "os": "linux",
+              "tags": {
+                "fedora-29-helix-amd64-$(System:DockerfileGitCommitSha)-$(System:TimeStamp)": {}
```
Comment (MichaelSimons) [inline manifest.json]: The Docker convention is that `amd64` is implied if the tag doesn't include an architecture.  For consistency reason, please remove from the tag.

File: src/fedora/29/helix/amd64/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,60 @@
+FROM fedora:29
```
Comment (MichaelSimons) [inline src/fedora/29/helix/amd64/Dockerfile]: I'm curious why this Dockerfile isn't based on the image produced from the `src/fedora/29/Dockerfile`?  Are there scenarios in which the dependencies will diverge?  I naively would think we would want to automatically get any changes the product teams make in `src/fedora/29/Dockerfile`.

Comment (MattGal) [reply src/fedora/29/helix/amd64/Dockerfile]: No good reasons, wasn't sure how to do that.

**Comment by MichaelSimons** [comment]
The following images were produced as a result of this PR:

mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-29-3d9d2e3-20190225225637
mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-29-helix-3d9d2e3-20190225225637

**Pull request merged**